.github/ISSUE_TEMPLATE/contribute.txt

```diff
@@ -0,0 +1,24 @@
+---
+name: Contribute
+about: Let us know you have an example you'd like to contribute
+
```

Comment:
These two lines are unclear to me (as full line -- but see below):
* Should they leave "Contribute" as the name of this file or do they replace with a name for their example? 
* How does "Let us ..." relate to "About"? I'd have expected something like "Tell us about the project you wish to contribute." 
Also, "name" and "about" should be "Name" and "About" (again, see below)

Maybe the post-colon text pops up in GitHub by itself (and "name" and "about" are just keywords)? If so, then just needs a period after "like to contribute"

Reply:
This is just GH's header in the markdown doc.